.github/workflows/ci-deploy.yml

```
@@ -112,4 +113,25 @@ jobs:
               state: 'success',
               environment: envName,
               environment_url: '${{ steps.deploy.outputs.url }}'
-            });
\ No newline at end of file
+            });
+
+  deploy_firestore_rules:
+    if: ${{ github.event_name == 'push' && github.ref_name == 'main' && secrets.FIREBASE_SERVICE_ACCOUNT != '' && secrets.FIREBASE_PROJECT_ID != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)